Tests/SwiftDocCUtilitiesTests/SignalTests.swift

```diff
@@ -32,32 +32,34 @@ class SignalTests: XCTestCase {
     /// `signal-test-app` sends a kill signal to itself and uses ``Signal`` to intercept that signal
     /// and set the exit code to 99.
     func testTrappingSignal() throws {
-        let signalTestAppPath = productDirectory.appendingPathComponent("signal-test-app").path
+        throw XCTSkip("This test hangs under tsan rdar://90715477")
```

Comment:
nit: Can we use try XCTSkipIf(true, ...) so that we don't need to comment out the body of the test.

That reduces the risk that refactoring renames a method that's used in commented out test code.

Reply:
Great point, I'll make that change.